bus-board/src/components/BusDataTable.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {IJsonBus} from "../modules/IJsonBus";
+import {BusDataRow} from "./BusDataRow";
+
+export const BusDataTable: React.FC<{ bussesData: IJsonBus[], buttonPressed: boolean }> = ({bussesData, buttonPressed}) => {
+
+    const rows: React.ReactElement[] = [];//scapam de any mai tarziu
+    if (!buttonPressed) {
+        return (
+            <div>Search a Stop Point to show available busses.</div>
+        )
+    }
+    if (bussesData.length === 0 || bussesData === undefined) {

```

Comment:
I think these should be reversed as the length one already considers bussesData to be defined. Also it seems buses doesn't double the s on plural.

Reply:
Reversed + spellchecked.